test/executor.test.js

```diff
@@ -754,6 +755,20 @@ describe('executor', function() {
       .get('/')
       .expect('passport', 'initialized', done);
   });
+
+  describe('when booting with env', function() {
+    it('should set the `booting` flag during execution', function(done) {
+      expect(app.booting).to.be.undefined();
+      boot.execute(app, envAppInstructions(), function(err) {
+        expect(err).to.be.undefined();
```

Comment:
Please use `if(err) return done(err)`, that produces more helpful assertion error message.